src/App.tsx

```diff
@@ -123,9 +139,6 @@ function CustomDrawerContent(props) {
           return <Text style={styles.icon}>&#xE713;</Text>;
         }}
       />
-      <ScrollView style={{height: '85%'}} {...props}>
```

Comment:
If we'd rearrange RNGalleryList to the order we want, is there anything stopping us from having 126-128 (or 130-132) be the sole logic for rendering the drawer?

Reply:
Extra logic accounts for hamburger menu staying visible when drawer opens and Home and Settings items staying in a static position as the drawer scrolls so they are always accessible.